static/upcomingScreeningsForMoviepage.js

```diff
@@ -25,13 +25,12 @@ document.addEventListener('DOMContentLoaded', async () => {
       screeningsData.data.forEach((screening) => {
         const listItem = document.createElement('li');
         const startTime = new Date(screening.attributes.start_time);
-        const formatDate = startTime.toLocaleDateString('sv-Se');
         const formatTime = startTime.toLocaleString('sv-SE', {
           timeZone: 'UTC',
         });
-        const formattedDateTime = `${formatDate}, ${formatTime}`;
+        const formattedDateTime = `${formatTime}`;
         headline.textContent = 'Kommande visningar';
-        listItem.textContent = `Datum: ${formattedDateTime}, Plats: ${screening.attributes.room}`;
+        listItem.textContent = `Datum: ${formattedDateTime} Plats: ${screening.attributes.room}`;
```

Comment:
You can if you like format your payload in the backend so that you dont have the need to use `screening.attributes.room`.

heres an exampel of my code and how ive done it.
`const reviews = payload.data.map((data) => ({`
    `id: data.id,`
    `reviewer: data.attributes.author,`
    `rating: data.attributes.rating,`
    `comment: data.attributes.comment,`
  `}));`

 ` const formattedRes = {`
    `review: reviews,`
    `pagination: payload.meta.pagination,`
  `};`

  `return formattedRes;`

Keep up the good work! 👍🏼